sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.institutions.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -65,21 +66,158 @@ describe("AssessmentInstitutionsController(e2e)-getAssessmentNOA", () => {
       },
     );
     await db.msfaaNumber.save(currentMSFAA);
-    const application = await saveFakeApplicationDisbursements(db.dataSource, {
-      institution: collegeF,
-      institutionLocation: collegeFLocation,
-      student,
-      msfaaNumber: currentMSFAA,
-    });
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        institution: collegeF,
+        institutionLocation: collegeFLocation,
+        student,
+        msfaaNumber: currentMSFAA,
+      },
+      {
+        offeringIntensity: OfferingIntensity.fullTime,
```

Comment:
since this test is for full-time, maybe we can add it to the test description to avoid confusion